src/common/utils/makeChartShareQuote.ts

```diff
@@ -33,5 +34,20 @@ export default function makeChartShareQuote(
       1,
     )} daily new cases per 100k population, according to @CovidActNow. See the chart: `;
   }
-  return `I'm keeping track of ${displayName}'s COVID data and risk level with @CovidActNow. What does your community look like?`;
+  return `I'm keeping track of ${displayName}'s vaccination progress and COVID risk level data with @CovidActNow. What does your community look like?`;
+}
+
+export function makeAddedChartShareQuote(
+  displayName: string,
+  metric: ExploreMetric,
+) {
+  if (metric === ExploreMetric.DEATHS) {
+    return `Daily COVID deaths in ${displayName}, according to @CovidActNow. See the chart: `;
```

Comment:
Right now we end up just sharing the URL for the location page and so "See the chart: " won't make sense...

Not sure what your plans are for chart sharing, but we'll need to either generate URLs and unique share images for all these charts or change this share quote text to make more sense given we're just sharing the location page.

Reply:
lowest lift to unblock shipping would be editing the share quote. so something more like 'Daily COVID deaths in ${displayName}, according to @CovidActNow. See the data:"

or maybe something more robust like 'Daily COVID deaths in ${displayName}, according to @CovidActNow. See ${displayName}'s vaccination progress and COVID risk level data: '

Reply:
I think it's weird to say "Daily COVID deaths in ${displayName}" but then the share image and the URL have nothing specifically to do with daily covid deaths.

How about:
> Check out Daily COVID deaths, vaccination progress, and other key metrics for ${displayName} at @CovidActNow:

Don't feel strongly though so feel free to make an executive decision and ship it.

Reply:
oo yes, going with that^